tasks/config.yml

```diff
@@ -28,11 +28,23 @@
     group: root
     mode: 0644
 
+- name: Create client-config-dir
+  file:
+    path: "{{ openvpn_base_dir }}/staticclients"
+    state: directory
+
+- name: Configure client-config-dir
```

Comment:
I don't think you're configuring the directory

Reply:
fixed